validator/src/commands/repair_whitelist/mod.rs

```diff
@@ -53,24 +88,29 @@ pub fn command(_default_args: &DefaultArgs) -> App<'_, '_> {
 pub fn execute(matches: &ArgMatches, ledger_path: &Path) -> Result<(), String> {
     match matches.subcommand() {
         ("get", Some(subcommand_matches)) => {
-            let output = OutputFormat::from_matches(subcommand_matches, "output", false);
+            let repair_whitelist_get_args =
+                RepairWhitelistGetArgs::from_clap_arg_match(subcommand_matches)?;
+
             let admin_client = admin_rpc_service::connect(ledger_path);
             let repair_whitelist = admin_rpc_service::runtime()
                 .block_on(async move { admin_client.await?.repair_whitelist().await })
                 .map_err(|err| format!("get repair whitelist request failed: {err}"))?;
 
-            println!("{}", output.formatted_string(&repair_whitelist));
+            println!(
+                "{}",
+                repair_whitelist_get_args
+                    .output
+                    .formatted_string(&repair_whitelist)
+            );
         }
         ("set", Some(subcommand_matches)) => {
-            let whitelist = if subcommand_matches.is_present("whitelist") {
-                let validators_set: HashSet<_> =
-                    values_t_or_exit!(subcommand_matches, "whitelist", Pubkey)
-                        .into_iter()
-                        .collect();
-                validators_set.into_iter().collect::<Vec<_>>()
-            } else {
+            let RepairWhitelistSetArgs { whitelist } =
+                RepairWhitelistSetArgs::from_clap_arg_match(subcommand_matches)?;
+
+            if whitelist.is_empty() {
```

Comment:
We do nothing if the repair whitelist is empty. So, we should make whitelist a required argument.

That being said, this was pre-existing behavior so I'm fine shipping the PR as-is, and will clean this up immediately after yours goes in